proto/types/receiptstatus.proto

```diff
@@ -34,6 +34,7 @@ enum ReceiptStatus {
   ErrUnauthorizedOperator           = 203;
   ErrInvalidBucketType              = 204;
   ErrCandidateNotExist              = 205;
-  ErrWithdrawBeforeUnstake          = 206;
-  ErrWithdrawBeforeMaturity         = 207;
+  ErrUnstakeBeforeMaturity          = 206;
```

Comment:
can you add 2 more:
ErrCandidateAlreadyExist (try to register an existing candidate)
ErrCandidateConflict (conflict with existing candidate)